dashboard/app/views/levels/show.html.haml

```diff
@@ -78,7 +78,8 @@
   -# special tracking image to make sure a cookie is set when hoc starts (/hoc/1)
   = image_tag(tracking_pixel_url(@script))
 
-= render partial: 'levels/admin'
+- if current_user && (!@level.uses_lab2? || (@level.uses_lab2? && !current_user.levelbuilder?))
```

Comment:
Just to confirm - since previously there are no checks here for `current_user` or `current_user.levelbuilder`, are we okay to add them here in addition to the lab2 check? Logically seems fine to me, but just want to double check if there are any other assumptions changing here (e.g. if there's another permission type that's able to view the extra links box)

Reply:
Yeah it should be fine. The first line of the haml file is
```
if current_user && (current_user.levelbuilder? || current_user.project_validator? || Experiment.get_editor_experiment(current_user))
```

So we were already gating on `current_user`. Then we either show the box if this isn't a lab2 level (same as before), or if it is lab2 and the user is not a levelbuilder (so we can show the links for project validators).